Scenes/Creeps/Creep.gd

```diff
@@ -439,12 +438,21 @@ func _calculate_path_length(path: Path2D) -> float:
 # it's z index so that the sprite is drawn correctly in
 # front of tiles.
 func _calculate_current_z_index() -> int:
-	var base_z_index: int = _path.get_z(_current_path_index)
 	var height: float = -_visual.position.y
-	var height_z_index = _map.world_height_to_z_index(height)
-	var total_z_index: int = base_z_index + height_z_index
 
-	return total_z_index
+# 	TODO: "100" is the placeholder. Figure out actual logic
```

Comment:
There should be no state in-between air and ground units. How ground creeps can have height more than 0?

Reply:
Staff of the Wild Equus item and Broken Circle of Wind tower use Creep.adjust_height() to raise the creep in the air.